src/index.css

```diff
@@ -1,8 +1,25 @@
-/* ./src/index.css */
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+
+.home-btn {
+  @apply bg-white  text-gray-900 w-40 py-2 font-light shadow-hover hover:text-gray-100 rounded-lg hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-600 focus:ring-opacity-50;
+}
+
+.nav-btn {
+  @apply bg-blue-600 text-gray-100 w-40 py-3 flex items-center rounded-lg justify-center shadow-bottom font-light hover:bg-blue-500 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:ring-opacity-50;
+}
```

Comment:
I see that you changed the button background colors to blue. eemm I dont really think the two colors go together. I think the previous shades of green we had is better. But I would like to know others think. I came across this tool for checking what colors to combine incase if we want to use a different color than green. [here](https://coolors.co/)

Reply:
No worries, I am still trying things out. Thanks for sharing the link. I know this side, but the problem would be, that there are too many different colors on the app than I guess. Yes, I was sticking to the colors of the home view, but of course, I want everyone to be happy with the design, so. I am sooo open to pink 😀 (no worries). I think I will make screenshots of different color combinations later today, so everyone can take a look, get used to the colors, and can choose which combination he/she prefers. Therefore, I will also try out a combination of the link you shared here 😊